src/main/kotlin/no/nav/syfo/consumer/pdfgen/PdfgenConsumer.kt

```diff
@@ -24,15 +24,23 @@ class PdfgenConsumer(urlEnv: UrlEnv, val pdlConsumer: PdlConsumer, val databaseI
 
     private val log = LoggerFactory.getLogger(PdfgenConsumer::class.qualifiedName)
 
-    suspend fun getMerVeiledningPDF(fnr: String, isBrukerReservert: Boolean): ByteArray? {
+    suspend fun getMerVeiledningPdfForReserverte(fnr: String): ByteArray? {
```

Comment:
Ser at vi ofte definerer url som variabel, bør kanskeje være i samme stil. Littmer esbar også. Gjelder `getMerVeiledningPdfForReserverte()` og `getMerVeiledningPdfForDigitale()`
Men not a big deal :)

Reply:
Mener du miljøvariabler i nais-filene eller som konstanter i PdfgenConsumer?